src/main/java/domain/CardSetting.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class CardSetting {
+    private static final List<TrumpCard> CARD_DECK = new ArrayList<>();
+
+    public static void initCache() {
+        Arrays.stream(CardShape.values())
+                .forEach(cardShape -> Arrays.stream(CardNumber.values())
+                        .forEach(cardNumber -> CARD_DECK.add(new TrumpCard(cardShape, cardNumber))));
+        Collections.shuffle(CARD_DECK);
+    }
```

Comment:
클래스가 생성됌과 동시에 static 변수의 초기화가 빌요하다면 `static 블록`에서 초기화를 해줄 수 있을 것 같아요. 

```suggestion
    static {
        Arrays.stream(CardShape.values())
            .forEach(cardShape -> Arrays.stream(CardNumber.values())
                .forEach(cardNumber -> CARD_DECK.add(new TrumpCard(cardShape, cardNumber))));
        Collections.shuffle(CARD_DECK);
    }
```

Reply:
사실 처음에는 static 블록을 사용해서 초기화했었는데 그렇게 하니 테스트 코드를 전체 실행 시켰을 때 해당 데이터를 공유해버리는 문제가 발생했습니다. 
이를 해결하기 위해서 정적 메서드로 분리 후 @BeforeEach로 삭제->초기화 시켜주는 과정을 추가했습니다.

ac44d557121a3c4337d05594c049a3c2a63125f9